.github/workflows/nightly-test.yaml

```diff
@@ -0,0 +1,47 @@
+name: Nightly Latest/Edge Tests
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs every midnight
+
+jobs:
+  test-integration:
+    name: Integration Test ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.releases }}
+    strategy:
+      matrix:
+        os: ["ubuntu:20.04", "ubuntu:22.04"]
+        arch: ["amd64", "arm64"]
+        releases: ["latest/edge", "1.30-classic/beta"]
```

Comment:
lets get this going with one branch first .. reduce this to a list of one "latest/edge"